fprintd 1.92.0 fails to build on latest unstable

### Describe the bug

Package fprintd doesn't build with the latest unstable (currently on commit https://github.com/NixOS/nixpkgs/commit/5b091d4fbe3b7b7493c3b46fe0842e4b30ea24b3).

### Steps To Reproduce

With the following configuration, using ThinkPad X1 Carbon 7th gen:

```nix
security = {
  pam = {
    services = {
      login.fprintAuth = true;
      swaylock-effects.fprintAuth = true;
      sudo.fprintAuth = true;
      system-local-login.fprintAuth = true;
      su.fprintAuth = true;
    };
  };
};

services.fprintd.enable = true;
```

After a `nix flake update` to the latest unstable, the compilation gives an error:

```
error: builder for '/nix/store/8gz3dz8h1yais5vk129xm0wnpln6n0wy-fprintd-1.92.0.drv' failed with exit code 1;
       last 10 log lines:
       > Expected Fail:      0
       > Fail:               0
       > Unexpected Pass:    0
       > Skipped:            0
       > Timeout:            11
       >
       > Full log written to /build/source/build/meson-logs/testlog.txt
       > FAILED: meson-test
       > /nix/store/cm0di021fzm7kay4q262p23j8bnp5f8z-meson-0.57.1/bin/meson test --no-rebuild --print-errorlogs
       > ninja: build stopped: subcommand failed.
       For full logs, run 'nix log /nix/store/8gz3dz8h1yais5vk129xm0wnpln6n0wy-fprintd-1.92.0.drv'.
error: 1 dependencies of derivation '/nix/store/pg6gk20xxh6hm5pa1a64i46gaq43iq6x-chfn.pam.drv' failed to build
error: 1 dependencies of derivation '/nix/store/9xxjcr38fiywsmp6ak1b4m6ilcqfdq8y-dbus-1.drv' failed to build
error: 1 dependencies of derivation '/nix/store/j07ga9k17agqvlw0gvy9yr7zx80jan2r-system-path.drv' failed to build
error: 1 dependencies of derivation '/nix/store/34yadjxgqpwglaxhqmnnf9rjcbh9n6g6-nixos-system-purrpurr-22.05.20211230.5b091d4.drv' failed to build
```

### Notify maintainers

@abbradar @grahamc

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.10, NixOS, 22.05 (Quokka)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.5.1`
 - channels(root): `"nixos-21.11pre294109.fbfb79400a0"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

--- a/pkgs/development/libraries/libfprint/default.nix
+++ b/pkgs/development/libraries/libfprint/default.nix
@@ -19,7 +19,7 @@
 
 stdenv.mkDerivation rec {
   pname = "libfprint";
-  version = "1.92.1";
+  version = "1.94.1";
   outputs = [ "out" "devdoc" ];
 
   src = fetchFromGitLab {
@@ -27,7 +27,7 @@ stdenv.mkDerivation rec {
     owner = "libfprint";
     repo = pname;
     rev = "v${version}";
-    sha256 = "0dpwzmwl9jjpaz44znvy3v8s9sln0c71b756rs1knk0zx8sa1qbc";
+    sha256 = "sha256-xFmby1x2TRZqXrV9Einqu3002qMAN5tQga2mIAHfC9c=";
   };
 
   nativeBuildInputs = [

--- a/pkgs/tools/security/fprintd/default.nix
+++ b/pkgs/tools/security/fprintd/default.nix
@@ -25,7 +25,7 @@
 
 stdenv.mkDerivation rec {
   pname = "fprintd";
-  version = "1.92.0";
+  version = "1.94.1";
   outputs = [ "out" "devdoc" ];
 
   src = fetchFromGitLab {
@@ -33,7 +33,7 @@ stdenv.mkDerivation rec {
     owner = "libfprint";
     repo = pname;
     rev = "v${version}";
-    sha256 = "0bqzxxb5iq3pdwdv1k8wsx3alirbjla6zgcki55b5p6mzrvk781x";
+    sha256 = "sha256-XHfHPffVp0jV3Md9Gui9v/nyOJ/bTWM3+hiR7WdEsgQ=";
   };
 
   nativeBuildInputs = [

